quaint/src/connector.rs

```diff
@@ -10,7 +10,7 @@
 //! querying interface.
 
 mod connection_info;
-pub(crate) mod metrics;
+pub mod metrics;
```

Comment:
nitpick: we try to follow a convention for grouping imports and exports in the codebase. This should be grouped with the public exports. In general, it looks like this:

```rust
pub mod a;
pub mod b;

mod c;
mod d;

pub use c::Something;

use d::SomethingElse;

```